app/uk/gov/hmrc/tariffclassificationfrontend/controllers/CasesController.scala

```diff
@@ -32,7 +32,7 @@ class CasesController @Inject()(casesService: CasesService,
                                 implicit val appConfig: AppConfig) extends FrontendController with I18nSupport {
 
   def gateway: Action[AnyContent] = Action.async { implicit request =>
-    casesService.getAllCases.map { cases =>
+    casesService.getGatewayCases.map { cases =>
```

Comment:
Do we need one function for each queue?

Reply:
I'm thinking we'll eventually have a `Queue` class which we can just pass into a `casesService.getCases(queue)` method. We can refactor on the next queue frontend task :)